### DISCUSSION TITLE
Excelsior animation

### DISCUSSION THREAD

**Discussion by wofo99** [fedorovvl/tso_client#39] [Q&A] [answered]
Hi. I do not see any animations in excelsior if I spend some crystals and hit the GO-Button. Is this normal or am I doing something wrong? In client version 1.4.8 the oven canvas is just black, while in 1.4.9 at least the oven is visible, but still no animation after GO. Any hint is welcome :-)

**Comment by fedorovvl** [comment, marked as answer]
hi. its very old bug with excelsior in flash client. the main thing is that it works, the animation is not so important :)

**Reply by wofo99** [reply]
Thank you :-)
